Add OpenTicket backup instructions to update procedure (#3838)
centreon/centreon-documentation: versioned_docs/version-23.04/alerts-notifications/ticketing.md

## Configuration

> 

Author: ```suggestion
```

## Configuration

> > Warning! There is a chance of your custom configuration disappearing when updating Centreon. To avoid this issue, refer to our [Update Centreon](../update/update-centreon-platform.md) procedure.

Author: ```suggestion
> Warning! There is a chance of your custom configuration disappearing when updating Centreon. To avoid this issue, refer to our [Update Centreon](../update/update-centreon-platform.md#Perform-a-backup) procedure.
```

> > Warning! There is a chance of your custom configuration disappearing when updating Centreon. To avoid this issue, refer to our [Update Centreon](../update/update-centreon-platform.md#Perform-a-backup) procedure.

Author: ```suggestion
> Warning! Open Ticket users with a custom configuration should [perform a backup before updating Centreon](../update/update-centreon-platform.md#Perform-a-backup) to avoid them being erased during the update process.
```

> > Warning! Open Ticket users with a custom configuration should [perform a backup before updating Centreon](../update/update-centreon-platform.md#Perform-a-backup) to avoid them being erased during the update process.

Author: ```suggestion
```
